mara_google_sheet_downloader/mara_integration.py

```diff
@@ -90,8 +89,7 @@ def _invocation(use_flask):
     import mara_google_sheet_downloader.__main__
     main_module = mara_google_sheet_downloader.__name__
     flask_command_name = mara_google_sheet_downloader.__main__.gs_download_to_csv.callback.__name__.replace('_', '-')
-    python = sys.executable
-    invocation = f'flask {main_module}.{flask_command_name}' if use_flask else f'{python} -m {main_module}'
```

Comment:
I would keep this as is, just to be a bit safer against not activated virtualenv: I'm currently unsure what would happen with your version.

Reply:
I guess it would not work either since module `mara_google_sheet_downloader` would not be available. But I might be wrong here.

I tested it and it works. I like it since it reduces the shell command length.
When you would install your module without venv, you would get a global shell entry point and it would still work, so, I don't really get your point but if you insist I can remove it.

Reply:
I mean this: 
* Installing all into a `.venv`
* NOT calling `.venv/bin/activate`
* `.venv/bin/flask mara_pipelines.ui.run ...`

This works with the current setup but my understanding is not with the new style.

Reply:
... yea, but this will not fix when you use the `use_flask_command` argument in the `DownloadGoogleSpreadsheet` command. This will still fail because it uses `flask` directly.

Frankly, just that you can call `.venv/bin/flask ...` doesn't mean that this is a good idea. Mabe I don't undestand the best practice here how to call mara but it just sounds strange to me. I thought mara is a shell execution tooling, so, when you really need that, wouldn't it then make more sence to install the module globally so that you can make the call `mara-google-sheet-downloader --params ....`? (that would make it work, but might not be a good idea either depending on your environment).
I have the feeling that we really need a best practice documentation how to use/run mara....

I added now a new commit which reverts this change so you can choose what you want to merge: the whole branch or just the first commit.

Reply:
I complete agree with you that starting the mara / flask thingy without an activated env is a bad idea, but I try to be conservative and make it possible (and I sure used it that way, despite usually having an autoenv activating the venv when I cd into the dir). 

IMO it's usually also a good idea to use the `<python> -m <module>` way as that ensures that you do not get screwed by (accidentally) not installed tools in the venv vs in system. E.g. `pip ...` will happily fall back to the system pip, but `python3 -m pip` will either install into the current activated venv or fail (if no pip is installed). Or a `mara-google-sheet-downloader` installed as system which by accident is available and the venv version not.